config/crd/observability.giantswarm.io_grafanaorganizations.yaml

```diff
@@ -104,8 +122,6 @@ spec:
                 description: OrgID is the actual organisation ID in grafana.
                 format: int64
                 type: integer
-            required:
```

Comment:
What happens to the CRs that have this property when we deploy CRDs that delete the property?
Do the CRs get deleted, or (as it's in the `status` part) they get silently updated?

Reply:
oh, it's just removed from `requiered` but is still valid! Got it! :+1:

Reply:
Yes it still is :)